src/material/stepper/step-header.ts

```diff
@@ -35,7 +35,6 @@ import {_CdkPrivateStyleLoader, _VisuallyHiddenLoader} from '@angular/cdk/privat
   host: {
     'class': 'mat-step-header',
     '[class]': '"mat-" + (color || "primary")',
-    'role': 'tab',
```

Comment:
Just FYI: this will inherit `role="button"` from the CDK. Is that intentional or did you mean to remove the role altogether?